src/components/text/Text.tsx

```diff
@@ -27,12 +17,37 @@ const Text = ({ children, weight = 'regular', style, ...props }: Props) => {
   const { neutral } = useThemeColors()
   return (
     <RNText
-      style={[{ color: neutral, fontFamily: fontByWeight[weight] }, style]}
+      style={[{ color: neutral, ...fontWeight(weight) }, style]}
       {...props}
     >
       {children}
     </RNText>
   )
 }
 
+type AnimatedProps = {
+  children: React.ReactNode
+  weight?: keyof typeof fontByWeight
+} & Parameters<typeof Animated.Text>[0]
```

Comment:
just curious if there is a better way to get the types for animated here

Reply:
I don't think there's a clean way to get the props of an Animated component. This also works apparently:

```
Animated.AnimatedProps<React.ComponentPropsWithRef<typeof RNText>>
```